tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -55,12 +55,14 @@ def test_do_not_inhibit_if_module_is_not_loaded(shell, convert2rhel):
     """
     assert shell("modprobe -r -v bonding").returncode == 0
 
-    if "oracle-7" in system_version or "centos-7" in system_version:
+    if "oracle-7" in system_version:
         prompt_amount = 3
+    elif "centos-7" in system_version:
+        prompt_amount = 4
```

Comment:
@danmyway, @kokesak, I needed to bump up the number due to an additional y/n question related to the python-syspurpose package installed under the other tier0 test:
https://github.com/oamg/convert2rhel/blob/v1.0/tests/integration/tier0/test-check-rollback-handling/test_check_rollback_handling.py#L9

It's quite weird. Shouldn't the previous test "clean" the system to not influence this other test?

Reply:
In the last run we are getting past the PONR, so I guess no need to bump up the `prompt_amount` 
```
20:45:34                 out: Successfully validated the yum transaction set.
20:45:34                 out: WARNING - ********************************************************
20:45:34                 out: WARNING - The tool allows rollback of any action until this point.
20:45:34                 out: WARNING - By continuing all further changes on the system will need to be reverted manually by the user, if necessary.
20:45:34                 out: WARNING - ********************************************************
20:45:34                 out: 
20:45:35                 out: Continue with the system conversion? [y/n]: y
```